cypress/integration/editor_spec.js

```diff
@@ -18,11 +18,22 @@ context(
     it("Show a modal to rename the project", () => {
       cy.get(".sc-eJwWfJ.hZFFnD").click();
 
-      cy.get("h2").should("contain.text", "Name Your Project");
+      cy.get('[data-testid="modal"]').contains("Name Your Project", { matchCase: false });
```

Comment:
**Rocking:** I like your approach of choosing case insensitivity here over modifying the string.

**Observation**: There's nothing technically wrong with this approach. My personal style is to make a specific assertion over letting the `contains()` call pass/fail, but the end result is the same.